src/gui/optionsdialog.ui

```diff
@@ -839,6 +839,56 @@
                  </property>
                 </widget>
                </item>
+               <item>
+                <layout class="QHBoxLayout" name="stopConditionLayout">
+                 <item>
+                  <widget class="QLabel" name="stopConditionLabel">
+                   <property name="text">
+                    <string>Torrent stop condition:</string>
+                   </property>
+                  </widget>
+                 </item>
+                 <item>
+                  <widget class="QComboBox" name="stopConditionComboBox">
+                   <property name="currentIndex">
+                    <number>0</number>
+                   </property>
+                   <property name="toolTip">
+                    <string>Metadata received: Torrent will stop after metadata is received. Torrents that have metadata initially aren't affected.
+Files checked: Torrent will stop after files are initially checked. This will also download metadata if it wasn't there initially.</string>
+                   </property>
+                   <item>
+                    <property name="text">
+                     <string>None</string>
+                    </property>
+                   </item>
+                   <item>
+                    <property name="text">
+                     <string>Metadata received</string>
```

Comment:
Maybe use: 
```suggestion
                     <string>After metadata received</string>
```

Reply:
I would prefer to keep it as is for the start (to save some space). I believe it is understandable enough (especially with tool tip available).